### PR TITLE
feature/AT-SCE-UI_Readme - US 030-UI-SCE: README File

### PR DESCRIPTION
**Pull request by moselGlez**
Added README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+
+## Tools
+
+| Technology | Purpose |
+| --- | --- |
+| JavaScript | Programming Lenguague |
+| Handlebars | JavaScript Templates |
+| Mocha & Chai | Unit Testing |
+| MongoDB | NoSQL Database |
+| MongoCompass | MongoDB GUI |
```
Comment (at-internship) [inline README.md]: MongoDBCompass | MongoDB GUI |

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+| Heroku | Platform as a Service |
+| Axios | Services Manage |
+
+## Contributor Team
+
+_Irving - Ghostbusters Team_
```
Comment (at-internship) [inline README.md]: GoF Team
KakashiTeam7

**Review by at-internship — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,85 @@
+# at-sce-ui
+
 # Node JS Project for AT Software Cost Estimation UI (Internal app)
+
+at-sce-ui app, is one of the 3 projects of this internship program, which is in charge of sce auth.
+
+GitHub - https://github.com/at-internship/at-sce-ui
+Heroku App - https://at-sce-ui.herokuapp.com/
```
Comment (at-internship) [inline README.md]: Heroku App PROD - https://at-sce-ui.herokuapp.com/
Heroku App QA - https://at-sce-ui-qa.herokuapp.com/

**Review by Emiliano-Plascencia — APPROVED** [review]

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Review by DianaArceoSuarez — APPROVED** [review]

**Pull request merged**